src/main/java/frc/trigon/robot/utilities/Conversions.java

```diff
@@ -0,0 +1,65 @@
+package frc.trigon.robot.utilities;
+
+public class Conversions {
+
+    private static final double maxMagTicks = 4096;
+    private static final double maxFalconTicks = 2048;
+
+    public static double magToDegrees(double magTicks) {
+        return magToRotations(magTicks) * 360;
+    }
+
+    public static double degreesToMag(double degrees) {
+        return degreesToRotations(degrees) *maxMagTicks;
+    }
+
+    public static double degreesToRotations(double degrees) {
+        return degrees / 360;
+    }
+
+    public static double rotationToDegrees(double rotations) {
+        return rotations * 360;
+    }
+
+    public static double rotationsToMag(double rotations) {
+        return rotations * maxMagTicks;
+    }
+
+    public static double magToRotations(double magTicks) {
+        return magTicks / maxMagTicks;
+    }
+
+    public static double falconToSeconds(double velocity) {
```

Comment:
falconTicks
+you can't convert distance to time

Reply:
f= ma^e